battle-service/src/main/java/org/astashonok/battleservice/services/impl/AuthUserServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package org.astashonok.battleservice.services.impl;
+
+import org.astashonok.battleservice.services.AuthUserService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthUserServiceImpl implements AuthUserService {
```

Comment:
this should be not a service but more like utility class, due to user id you will get from spring security context